lms/djangoapps/discussion_api/views.py

```diff
@@ -22,7 +22,7 @@
     get_thread_list,
     update_comment,
     update_thread,
-)
+    get_thread)
```

Comment:
This needs to come before `get_thread_list`. https://github.com/edx/edx-platform/wiki/Python-Guidelines#imports-order